spring-integration-jdbc/src/test/java/org/springframework/integration/jdbc/mysql/MySqlJdbcMessageStoreMultipleChannelTests.java

```diff
@@ -94,31 +92,25 @@ public void beforeTest() {
 
 	@After
 	public void afterTest() {
-		new TransactionTemplate(this.transactionManager).execute(new TransactionCallback<Void>() {
-			@Override
-			public Void doInTransaction(TransactionStatus status) {
-				final int deletedGroupToMessageRows = jdbcTemplate.update("delete from INT_GROUP_TO_MESSAGE");
-				final int deletedMessages = jdbcTemplate.update("delete from INT_MESSAGE");
-				final int deletedMessageGroups = jdbcTemplate.update("delete from INT_MESSAGE_GROUP");
-
-				LOG.info(String.format("Cleaning Database - Deleted Messages: %s, " +
-						"Deleted GroupToMessage Rows: %s, Deleted Message Groups: %s",
-						deletedMessages, deletedGroupToMessageRows, deletedMessageGroups));
-
-				return null;
-			}
+		new TransactionTemplate(this.transactionManager).execute(status -> {
+			final int deletedGroupToMessageRows = jdbcTemplate.update("delete from INT_GROUP_TO_MESSAGE");
+			final int deletedMessages = jdbcTemplate.update("delete from INT_MESSAGE");
+			final int deletedMessageGroups = jdbcTemplate.update("delete from INT_MESSAGE_GROUP");
+
+			LOG.info(String.format("Cleaning Database - Deleted Messages: %s, " +
```

Comment:
I'm against unconditional logs in tests.
Would you mind removing it in the next polishing?
Thanks